.ruby-version

```diff
@@ -1 +1 @@
-2.5.1
+2.6.3
```

Comment:
I couldn't install bundler in my laptop at the first attempt so I figured that using an up-to-date version is better.